lib/uri/common.rb

```diff
@@ -185,23 +185,13 @@ def self.parse(uri)
     RFC3986_PARSER.parse(uri)
   end
 
+  # Recursively merges the given URI strings +str+
```

Comment:
I think "recursively" is an odd word to use here because there's no recursion. I think it would be less confusing if we just dropped that word?

Reply:
Fixed.